### DISCUSSION TITLE
codegeex pro询问问题报错

### DISCUSSION THREAD

**Discussion by zybzbk** [THUDM/CodeGeeX2#261] [Q&A]
当我向codegeex pro询问问题时报错，报错内容为Failed to fetch

**Comment by crazygo** [comment]
你好，这个错误提示一般是因为网络不好导致的。请检查一下 IDE 是否设置了代理，是否在远程模式下工作。
如仍有问题，也欢迎添加我的微信 `nob301` 交流使用经验和反馈问题，谢谢。
